core/market/resolver/src/flatten.rs

```diff
@@ -1,29 +1,39 @@
 use serde_json::{json, Map, Value};
 
+#[derive(thiserror::Error, Debug)]
+#[error("Flattened JSON should be object, but found: {0}")]
+pub struct JsonObjectExpected(String);
```

Comment:
This will log a lot of spam on console.

Reply:
And it should,

Reply:
Why should it spam in your opinion?
You can always log on trace level whole json when this error occurs, but add only type of json entry to error message.

Reply:
bc if someone is using the wrong properties format, he should see what is causing the problem. It will not be spam. It is essential to help fixing the issue